src/components/UI/molecules/ArticleCard/ArticleCard.jsx

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-use-before-define */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import FlexContainer from '<atoms>/layouts/FlexContainer/FlexContainer';
+import Image from '<atoms>/Image/Image';
+import Text from '<atoms>/Text/Text';
+import Title from '<atoms>/Title/Title';
+import Divider from '<atoms>/Divider/Divider';
+import StarRating from '<molecules>/StarRating/StarRating';
+
+const ArticleCard = ({
+  imageUrl,
+  title,
+  description,
+  rating,
+  author,
+}) => {
+  return (
+    <ArticleCard.Container>
+      <FlexContainer
+        boxShadow="articleCard"
+        containerWidth='articleCardWidth'
+        alignItems='center'
+        justifyContent = 'center'
+      >
+        {renderImage()}
+        {title && (renderTitle())}
+        {description && (renderText())}
+        {renderDivider()}
+        {author && (renderAuthor())}
+        {renderRating()}
+      </FlexContainer>
+    </ArticleCard.Container>
+  );
+
+  function renderDivider() {
+    return (
+      <Divider
+        width="mds"
+      />
+    );
+  }
+
+  function renderImage() {
+    return (
+      <Image
+        imageUrl={imageUrl}
+        width='288px'
```

Comment:
Does this width take into account all cards in the codebase ?

Reply:
yeah